user-service/controller/user-controller.js

```diff
@@ -25,6 +27,20 @@ export async function createUser(req, res) {
     }
 }
 
+export async function deleteUser(req, res) {
+    try {
+        const { username } = req.params;
+        const resp = _deleteUser(username);

```

Comment:
Is the check automatically done for the existence of user name ? since the username is part of the url link

Reply:
Yup, we have the middleware to check the jwt and there is a check in the user-orm to check the existence of user.